### PR TITLE
move repo to filecoin-station/desktop

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

File: docs/DEVELOPING.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ graph
...
-1. `saturn-webui` publishes a new release with WebUI assets
-2. `saturn-l2` is updated to use the new `saturn-webui` version
-3. `saturn-l2` publishes a new release
-4. `filecoin-station` is updated to use the new `saturn-l2 `version
-5. `filecoin-station` publishes a new release
+1. `filecoin-project/saturn-webui` publishes a new release with WebUI assets
+2. `filecoin-saturn/L2-node` is updated to use the new `saturn-webui` version
+3. `filecoin-saturn/L2-node` publishes a new release
+4. `filecoin-station/desktop` is updated to use the new `L2-node` version
+5. `filecoin-station/desktop` publishes a new release
```
Comment (bajtos) [inline docs/DEVELOPING.md:36]: I am not sure if L2-Node is still bundling webui. These instructions may be outdated. Fixing them is probably out of the scope of this pull request, though.

**Review by bajtos — APPROVED** [review]
LGTM.

What is the impact on electron-autoupdater? Will the GH Releases URL used by the existing Station installations keep working after the change? (I mean, will GitHub redirect it?)

**Comment by juliangruber** [comment]
> What is the impact on electron-autoupdater? Will the GH Releases URL used by the existing Station installations keep working after the change? (I mean, will GitHub redirect it?)

Yes this is how it should work. See for example this URL, redirecting from the old to the new L2 repo: https://github.com/filecoin-project/saturn-l2/releases/tag/v0.4.5

File: docs/DEVELOPING.md (hunk 1 above)
Comment (juliangruber) [reply docs/DEVELOPING.md:36]: I agree, I wanted to keep the scope as small as possible

**Pull request merged**